profileservice/api/controllers/configs.py

```diff
@@ -27,10 +27,12 @@
 FLASK_ENV = os.getenv('FLASK_ENV', 'production')
 API_LOC = os.getenv('API_LOC', '../')
 PROFILE_URL_PREFIX = os.getenv('PROFILE_URL_PREFIX', '')
+SHIB_HOST = os.getenv('SHIBBOLETH_HOST', '')
+AUTH_ISSUER = os.getenv('AUTH_ISSUER', '')
 DEBUG = bool(os.getenv('DEBUG', 'False') == 'True')
 
-PROFILE_DB_NAME = 'profiledb'
-PII_DB_NAME = 'piidb'
+PROFILE_DB_NAME = os.getenv('PROFILE_DB_NAME', 'profiledb')
+PII_DB_NAME = os.getenv('PROFILE_DB_NAME', 'piidb')
```

Comment:
```suggestion
PII_DB_NAME = os.getenv('PII_DB_NAME', 'piidb')
```
To differentiate between Non PII and PII DB names.